src/github.rs

```diff
@@ -763,6 +778,12 @@ impl Issue {
     }
 }
 
+#[derive(Debug, serde::Deserialize)]
+pub struct Commit {
```

Comment:
Hm, any reason to not use `GithubCommit` here?

Reply:
Just didn't notice it existed, fixed.